lib/src/function-registry.ts

```diff
@@ -57,11 +57,11 @@ export class FunctionRegistry<sync extends 'sync' | 'async'> {
   call(
     request: OutboundMessage.FunctionCallRequest
   ): PromiseOr<InboundMessage.FunctionCallResponse, sync> {
-    const protofier = new Protofier(this);
-    const fn = this.get(request);
-
     return catchOr(
       () => {
+        const protofier = new Protofier(this);
+        const fn = this.get(request);
```

Comment:
Why this change?

Reply:
Because the error of this.get is not catches and raised back to compiler, the compilation will get stuck.